src/assets/scripts/client/aircraft/FlightManagementSystem/Fms.js

```diff
@@ -116,21 +116,11 @@ export default class Fms {
         */
         this._previousRouteSegments = [];
 
-        // TODO: Use this!
-        /**
-         * Airport the aircraft arrives to
-         *
-         * @for Fms
-         * @property arrivalAirport
-         * @type {AirportModel}
-         */
-        this.arrivalAirport = null;
-
```

Comment:
I was under the impression it wouldn't be "storing" the airport multiple times, but only pointers. But perhaps putting the airports in the navigation library as you suggest makes more sense, then the fms can access the information it will need.